recipes/lazrs-python/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "lazrs-python" %}
+{% set version = "0.4.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/lazrs/lazrs-{{ version }}.tar.gz
+  sha256: 1ddb597bf5f42eb2977a8eb8ae08d0b3c8106cace3665fc7e6aea8adda0b8e60
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('rust') }}
+  host:
+    - maturin
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - lazrs
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/lazrs/
+  summary: Python bindings for laz-rs
+  license: MIT
+  license_file: LICENSE.txt
+
```

```suggestion
  dev_url: https://github.com/laz-rs/laz-rs-python

```
Probably worth explicitly linking to the repo, either here or in the `home:` field